std/xml.d

```diff
@@ -2970,3 +2980,12 @@ private
         throw new XMLException(s);
     }
 }
+
+unittest
```

Comment:
I'm not sure about the location of this `unittest` in the file. Other more generic tests seem to be further up (lines [2716 through 2740](https://github.com/coderabhishek/phobos/blob/54ad8cbb9212373c1c5a17551b60cd1df8133db4/std/xml.d#L2716-L2740)).